src/helloworld/README.md

```diff
@@ -133,15 +133,16 @@ Breakpoint 1, main () at helloworld.c:6
 (gdb) 
 ```
 
-If you then step into the `printf` call you will see that GDB prints
-the capability argument with expanded information including bounds and
-permissions:
+If you then step, you into the `printf` (or `puts` depending on
+optimization level) call you will see that GDB prints the capability
+argument with expanded information including bounds and permissions:
 
 ```
 (gdb) s
-printf (fmt=0x1006e0 [rR,0x1006e0-0x1006ed] "Hello world\n")
-    at /home/bed22/cheri/cheribsd/lib/libc/stdio/printf.c:57
-57      /home/bed22/cheri/cheribsd/lib/libc/stdio/printf.c: No such file or directory.
+(gdb) s
```

Comment:
Is the intent to show both or just one? This has two steps

Reply:
Not quite enough replacement when pasting...